patchutils.go

```diff
@@ -225,8 +225,8 @@ func applyDiff(source string, diffFile *diff.FileDiff) (string, error) {
 			} else {
 				if line[1:] != sourceBody[currentOrgSourceI-1] {
 					return "", fmt.Errorf(
-						"content of original line in source (%d): %q and diff: %q aren't same",
-						currentOrgSourceI, line[1:], sourceBody[currentOrgSourceI-1])
+						"%w in source: (%d) %q and diff: %q",
```

Comment:
the nested error message should always come as the last part of fmt.Errorf. I'd use this: `"line %d in source (%q) and diff (%q): %w"`